NEWS.md

```diff
@@ -1,6 +1,54 @@
 JuMP release notes
 ==================
 
+Version 0.21 (Feb 13, 2020)
+-----------------------------
+
+Breaking changes:
+
+- Deprecated `with_optimizer` (#2090, #2084, #2141). You can replace
```

Comment:
We should provide a motivation for making this breaking change. I.e., in order to improve th econsistency of how solver parameters are specified...

Reply:
Thinking about this again, it's not necessary. We link to the PRs that have these discussions.